src/drawing/area.rs

```diff
@@ -466,9 +476,49 @@ impl<DB: DrawingBackend> DrawingArea<DB, Shift> {
             },
             backend: self.copy_backend_ref(),
             coord: Shift((self.rect.x0, self.rect.y0 + 10 + text_h as i32)),
+            inset: self.inset,
         })
     }
 
+    /// Alter area bounds
+    pub fn alter_diff(self, diff_tl: (i32,i32), diff_rb: (i32,i32)) -> Self {
```

Comment:
Could we just use `shrink` ? It looks a little bit redundant to me , but I may be wrong

Reply:
Sure, show me how :)